frontend/src/metabase-lib/v1/expressions/suggest.ts

```diff
@@ -40,7 +45,18 @@ const suggestionText = (func: MBQLClauseFunctionConfig) => {
   return displayName + suffix;
 };
 
-type SuggestArgs = {
+export const GROUPS = {
+  popularExpressions: {
+    displayName: t`Most used functions`,
```

Comment:
Clicking the group title causes the popover to disappear. Not critical, but if it's easy to fix we should do it.


https://github.com/metabase/metabase/assets/6830683/8dc58d79-b282-4bd7-9307-b9768d917a89